src/explore-education-statistics-common/src/modules/find-statistics/queries/tableBuilderQueries.ts

```diff
@@ -0,0 +1,22 @@
+import tableBuilderService from '@common/services/tableBuilderService';
+import { createQueryKeys } from '@lukemorales/query-key-factory';
+
+const tableBuilderQueries = createQueryKeys('tableBuilder', {
+  getGeoJson(
```

Comment:
We might not need this if we end up going down the combined query approach (outlined in my other comment), but if we do keep this, we should rename it to `getLocationGeoJson`